samples/root/Getting Started/index.html

```diff
@@ -28,11 +28,10 @@
         -->
         <p>
             <em>Brackets is a different type of editor.</em>
-            Brackets has some unique features like Quick Edit, Live Preview and others that you may not find in other 
-            editors. And Brackets is written in JavaScript, HTML and CSS. That means that most of you using Brackets 
-            have the skills necessary to modify and extend the editor. In fact, we use Brackets every day to build 
+            Brackets has some unique features like Quick Edit, Live Preview and others that you may not find in other
+            editors. And Brackets is written in JavaScript, HTML and CSS. That means that most of you using Brackets
+            have the skills necessary to modify and extend the editor. In fact, we use Brackets every day to build
             Brackets. To learn more about how to use the key features, read on.
-            
```

Comment:
Several other languages have this extra line also.  Do you want to remove it in all of them for consistency?  (Please squash commits if you do, for history cleanliness).